Scouting/src/main/java/ca/team2706/scouting/mcmergemanager/backend/BlueAllianceUtils.java

```diff
@@ -100,6 +100,39 @@ public void run() {
         }.start();
     }
 
+    // TODO This needs to be finished, Untested
+    // Gets the list of all teams attending the event
+    // This is necessary because the other fetch method does not return a string.
+    public static String fetchTeamsRegisteredAtEventString() {
```

Comment:
Since this doesn't take an argument, which event is it querying for?
From the code below I see it's pulling from the properties / setting menu. Maybe mention in the comment?

Nice though!

Reply:
Oh yeah it does, I got a little side tracked with teaching Pop-Up's and the comment_bar branch stuff but I will definitely do that.